pyphare/pyphare/pharesee/run/run.py

```
@@ -103,30 +105,69 @@ def GetE(self, time, merged=False, interp="nearest", all_primal=True, **kwargs):
+        h = compute_hier_from(_compute_to_primal, hier_, value="rho")
+        return ScalarField(h)
+
+    def GetVi(self, time, merged=False, interp="nearest", all_primal=True, **kwargs):
+        if merged:
+            all_primal = False
         hier = self._get_hierarchy(time, "ions_bulkVelocity.h5", **kwargs)
+
+        if not all_primal:
+            return self._get(hier, time, merged, interp)
+
+        h = compute_hier_from(_compute_to_primal, hier, x="Vx", y="Vy", z="Vz")
```

Ruff (0.14.14)

[error] Local variable `h` is assigned to but never used

Remove assignment to unused variable `h`

(F841)